src/unstract/sdk/llm.py

```diff
@@ -80,6 +81,19 @@ def complete(
         except Exception as e:
             raise parse_llm_err(e) from e
 
+    def stream_complete(
+        self,
+        prompt: str,
+        **kwargs: Any,
+    ) -> CompletionResponseGen:
```

Comment:
@harini-venkataraman actually, I suggest not exposing the llama index type if its possible. We'll have to add  it as a dependency if we want type hints

Reply:
Since it is a streaming API, the response is token by token. Hence typed it explicitly. Will revisit this.